code/controllers/subsystem/timer.dm

```diff
@@ -427,6 +429,10 @@ SUBSYSTEM_DEF(timer)
 	//get the list of buckets
 	var/list/bucket_list = SStimer.bucket_list
 
+	if(length(bucket_list) != BUCKET_LEN)
+		SStimer.reset_buckets()
+		bucket_list = SStimer.bucket_list
```

Comment:
This should not be done here, if it's really a problem for you then you need to fix this in init somewhere